frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -475,6 +484,22 @@ function ReaderPaging:onZoomModeUpdate(new_mode)
     self.zoom_mode = new_mode
 end
 
+function ReaderPaging:onZoomPanUpdate(settings)
+    local _settings = {
+        "zoom_pan_h_overlap",
+        "zoom_pan_v_overlap",
+        "zoom_pan_bottom_to_top",
+        "zoom_pan_direction_vertical",
+    }
+    for k, v in pairs(settings) do
+        if require("util").arrayContains(_settings, k) then
```

Comment:
Perhaps importing util at the top of the file would be better, as to avoid doing this several times (in a loop)